### PR TITLE
feat: show specific deploy error message

### PR DESCRIPTION
**Pull request by CeerDecy**
## Description
dice1.0 and dice2.0 : add deploy failcase to metaData

### CODE REVIEW

File: actions/dice/2.0/internal/pkg/dice/deploy/status_check.go [hunk 1]
```diff
@@ -202,13 +215,14 @@ func (d *deploy) check(drMap map[string]*common.DeployResult) (bool, error) {
 		utils.BatchPrintStatusCheckResult(statusResult)
 	}
 
+	// 以下代码在上面用defer执行
```
Comment (iutx) [inline actions/dice/2.0/internal/pkg/dice/deploy/status_check.go]: Remove unused code.

**Review by iutx — COMMENTED** [review]
@CeerDecy

File: actions/dice/2.0/internal/pkg/dice/deploy/status_check.go (hunk 1 above)
Comment (CeerDecy) [reply actions/dice/2.0/internal/pkg/dice/deploy/status_check.go]: @iutx done

File: actions/dice/2.0/internal/pkg/dice/deploy/status_check.go [hunk 2]
```diff
@@ -52,6 +53,7 @@ func (d *deploy) StatusCheck(orderId string, result map[string]*common.DeployRes
 		return nil
 	default:
 		// loop check deployments status
+		logrus.Infof("====> default")
```
Comment (iutx) [inline actions/dice/2.0/internal/pkg/dice/deploy/status_check.go]: @CeerDecy cc

**Review by iutx — DISMISSED** [review]
LGTM @sfwn 
[related pr](https://github.com/erda-project/erda/pull/6169)

File: actions/dice/2.0/internal/pkg/dice/deploy/status_check.go [hunk 3]
```diff
@@ -75,7 +76,7 @@ func (d *deploy) statusCheckLoop(ctx context.Context, drMap map[string]*common.D
 		case <-ticker.C:
 			isDeploying, err := d.check(drMap)
 			if err != nil {
-				logrus.Error("failed to check deploy", err)
+				logrus.Error("failed to check deploy ", err)
```
Comment (sfwn) [inline actions/dice/2.0/internal/pkg/dice/deploy/status_check.go]: use `Errorf`

**Review by sfwn — CHANGES_REQUESTED** [review]

**Review by sfwn — APPROVED** [review]

**Pull request merged**